pkg/cmd/server/kubernetes/master_config.go

```diff
@@ -43,7 +43,7 @@ import (
 )
 
 // AdmissionPlugins is the full list of admission control plugins to enable in the order they must run
-var AdmissionPlugins = []string{"NamespaceLifecycle", "OriginPodNodeEnvironment", "LimitRanger", "ServiceAccount", "SecurityContextConstraint", "BuildDefaults", "BuildOverrides", "ResourceQuota", "SCCExecRestrictions"}
+var AdmissionPlugins = []string{"NamespaceLifecycle", "OriginPodNodeEnvironment", "ClusterResourceOverride", "LimitRanger", "ServiceAccount", "SecurityContextConstraint", "BuildDefaults", "BuildOverrides", "ResourceQuota", "SCCExecRestrictions"}
```

Comment:
> @deads2k should this be the sole constant in the list?

For now, yes.  They should all be updated, but yours has more referrers than the rest so I'd like it to change now.  I don't think you should have to change them all in this pull, just make an issues and we'll followup.